src/components/Header/Header.stories.tsx

```diff
@@ -15,20 +15,14 @@
 
 import React from 'react';
 
-import Hamburger from '../Hamburger';
-
 import Header from '.';
 
 export default {
   title: 'Components/Header',
   component: Header,
 };
 
-export const Base = (args) => (
-  <Header {...args}>
-    <Hamburger size="kilo" />
-  </Header>
-);
```

Comment:
What's your intention behind removing the hamburger here? 

The stories also serve as documentation and I believe it's important to document the use case with the hamburger.

Reply:
I will revert 🙂